microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -70,164 +71,274 @@ After doing the above steps, the extension is ready to be submitted via Partner
+| **Website URL** | - | A webpage that provides additional information about your extension.  The `Website URL` must point to a webpage on your own website, not the web listing for your extension in the [Microsoft Edge Add-ons website](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home).  The `Website URL` helps users learn more about your extension, its features, and any other relevant information. |
+| **Support contact details** | - | The URL to your support webpage, or the email address to contact your support team. |
+| **Mature content** | - | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, see [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
 
-Select **Save & Continue** to continue to the **Store listings** section.
+Properties that are not indicated as Required are Optional.
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Your developer/organization name, **Website URL**, and **Support contact details** that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+
+After filling in the **Properties** page for the extension, click the **Save & Continue** button.
+
+The **Store listings** page opens.
 
 
 <!-- ========================================================================== -->
 ## Step 5: Add Store listing details for your extension
 
-The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.  To list your extension in the store, the following details are required.
+1. In the **Store Listings** page, click the **Edit details** button:
+
+![Store listing section for an extension](./publish-extension-images/store-listing-section-for-extension.png)
+
+
+The information provided in the **Store listings** page is displayed to users who view your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
```

Comment:
```suggestion
The information provided in the **Store Listings** page is displayed to users who view your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
```

Reply:
rendered, to look at pngs:
https://github.com/MicrosoftDocs/edge-developer/blob/bhuvanapriyap-patch-2/microsoft-edge/extensions-chromium/publish/publish-extension.md#step-5-add-store-listing-details-for-your-extension

Reply:
inconsistent ui at P. C.:
left nav: listings
top of page: Listings.
@captainbrosset - a UI bug.  trends in several companies would go w/ lowercase for the page title. 
Changed all 2 instances to uppercase L.  That's ok b/c they are not specifically saying "click the words _x_ in the left nav."